### PR TITLE
Fix form

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

**Comment by pypingou** [comment]
There are tests for this and testing it some more would imply testing it with multiple/different version of flask-wtf, not sure how easy this would be :s

**Comment by jeremycline** [comment]
It would be good to fix the existing broken test for travis, and it's easy to do a test matrix on different versions of the same library in Travis/Tox.

**Comment by pypingou** [comment]
Ok, so what started as: let's fix the tests ended up being: let's port to py3 :) 

At least, all tests are passing now!
https://travis-ci.org/fedora-infra/nuancier/builds/221669454

Re-review welcome :)

**Comment by sayanchowdhury** [comment]
:+1: for me

**Comment by pypingou** [comment]
@bowlofeggs and @jeremycline do you also want to have a look at it since you reviewed it before the port to py3 ? :)

File: nuancier/admin.py [hunk 1]
```diff
@@ -428,6 +428,6 @@ def admin_cache(election_id):
         LOG.debug('User: "%s" could not generate cache for "%s"',
                   flask.g.fas_user.username, election_id)
         LOG.exception(err)
-        flask.flash(err.message, 'error')
+        flask.flash('%s' % err, 'error')
```
Comment (jeremycline) [inline nuancier/admin.py]: Why not just use ``str(err)``?

**Comment by jeremycline** [comment]
I'd recommend breaking the Python 3 conversion out into a separate PR in the future, but I looked over it and it seems fine.

I would also recommend removing Python 3.4 and Python 3.5 from the "allowed_failures" dictionary in ``.travis.yml`` since they're now passing :smile:

**Comment by pypingou** [comment]
Squashed into 4 commits rather than 3 :)

**Comment by pypingou** [comment]
I'm going to assume you're ok with the squashing :)

**Pull request merged**